installer/templates/phx_web/components/core_components.ex

```diff
@@ -46,13 +46,15 @@ defmodule <%= @web_namespace %>.CoreComponents do
       role="alert"
       class={[
         "fixed top-2 right-2 mr-2 w-80 sm:w-96 z-50 rounded-lg p-3 ring-1",
-        @kind == :info && "bg-emerald-50 text-emerald-800 ring-emerald-500 fill-cyan-900",
+        @kind == :info && "bg-blue-50 text-blue-800 ring-blue-500 fill-blue-900",
```

Comment:
personally, I would use cyan or sky for info messages, but it may be bias from seeing so many UI Frameworks using the word "info" with a light blue color

Reply:
I guess sky it will be then :-)